src/prop-editor/prop-editor.scss

```diff
@@ -9,14 +9,22 @@
   .component-name {
     font-weight: bold;
     margin-bottom: $spacing;
+    padding-bottom: $spacing;
+    border-bottom: $gray3 1px solid;
   }
 
+  .reset {
+    width: calc(100% - 55px);
+  }
   .delete {
-    background-color: transparent;
-    border: none;
-    font-size: 24px;
-    margin-top: $spacing;
-    padding: 0;
+    width: 45px;
```

Comment:
How do you feel about keeping properties within a rule in alphabetical order?
Also keeping rules in alphabetical order?
I think it helps me find what I'm looking for faster.

Reply:
Thats fine, I can do that going forward.